paritytech/polkadot-sdk
# Es migration build

migrate jobs to gha
- build-linux-stable-cumulus
- build-test-parachain
- build-linux-stable
- build-test-collators
- build-malus
- build-linux-substrate
- prepare-bridges-zombienet-artifacts (might require help from zombienet team)
- build-push-image-polkadot-parachain-debug
- build-push-image-test-parachain
- build-push-image-polkadot-debug
- build-push-image-colander
- build-push-image-malus
- build-push-image-substrate-pr
- build-push-image-bridges-zombienet-tests

see [ci_cd#1013](https://github.com/paritytech/ci_cd/issues/1013)
